### ISSUE TITLE
sample cannot open

### ISSUE THREAD

**Issue by Restry** [jexp/neo4j-3d-force-graph#3]
beautiful 3d relationship maps , but the all sample can not open ,  would you help to check why ? it's seems like the datasource was use a local url . 

thanks

**Comment by jexp** [comment]
Yes you can have a local graph running with username neo4j and password test.
I guess I should update them for a remote test database.

**Comment by jexp** [comment]
Fixed, please check.
I now point them to a neo4j-cloud instance with the example data.

**Comment by Restry** [comment]
@jexp  thanks , i can open it now  :)

**Comment by spacemonqi** [comment]
Should it therefore be possible to clone the repo, and then run yarn install & yarn start as-is in order to render the game of thrones data? Because I am just getting the placeholder Joe & Jane.
Thanks in advance & apologies, I am quite new to this.

**Comment by jexp** [comment]
The renders are just inline pages that connect to fixed graphs. 

The react app should work with any db
